docs/bootnode.md

```diff
@@ -0,0 +1,25 @@
+# Bootnode
+
+The swap program uses a p2p network to discover offer-makers to do a swap with, and also the run the actual swap protocol. A node must know addresses of nodes already in the network to join the network. These nodes are often publicly posted and referred to as bootnodes. Bootnodes act as an entry-point into the p2p network. 
```

Comment:
I think the phrase "The swapd daemon" would be more obvious than "The swap program".

How are you editing this? Most people disable line-wrap in code editors and editing a 324 symbol line is unwieldy. Github will still free-flow the lines when viewed in the web browser if you add newlines. I'm been using an editor plugin that allows me to reformat documentation and code comments at 80 characters (a little narrower than our max line length for easier readability).